src/main/java/net/aufdemrand/denizen/events/player/PlayerStatisticIncrementsScriptEvent.java

```diff
@@ -61,7 +61,7 @@ public boolean matches(ScriptContainer scriptContainer, String s) {
         String lower = CoreUtilities.toLowerCase(s);
         String stat = CoreUtilities.getXthArg(2, lower);
 
-        if (!stat.equals(CoreUtilities.toLowerCase(statistic.toString())) && !stat.equals("increments")) {
+        if (!stat.equals("increments") && !stat.equals(CoreUtilities.toLowerCase(statistic.toString()))) {

```

Comment:
Rather than calling the lowercase first, it would check if stat equals "increments" first, slight optimisation? :)